wcivf/aws_lambda/handlers.py

```diff
@@ -24,7 +24,12 @@ def lambda_handler(event, context):
 
         Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
     """
-
+    print("EVENT BELOW")
+    print(event)
```

Comment:
By the way, did you know about `print(f"{event=}")`?

Reply:
Ah yes this would be a smarter way of doing it!